backend/kesaseteli/applications/api/v1/views.py

```diff
@@ -12,5 +14,23 @@ class ApplicationViewSet(AuditLoggingModelViewSet):
     )
```

Comment:
Does the queryset attribute need to remain defined, now that get_queryset() is defined? Can the select_related / prefetch_related be moved to get_queryset()?

Reply:
Yea I'll move the prefetching to `get_queryset` to make it more clear.